accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservice/impl/AceServiceImpl.java

```diff
@@ -591,7 +592,12 @@ private String purgeAuthorizables(Set<String> authorizableIds,
         try {
             JackrabbitSession js = (JackrabbitSession) session;
             UserManager userManager = js.getUserManager();
-            userManager.autoSave(false);
+            try {
+              userManager.autoSave(false);
+            } catch (final UnsupportedRepositoryOperationException e) {
```

Comment:
The latter would be even better as the javadoc states, that this method may throw this exception. That way we support also any other Jackrabbit implementations (although currently only JR2 and Oak exists, and for both the exception will not be thrown that way).